generator/generator.yml

```diff
@@ -361,3 +361,25 @@ modules:
       - picoMobileMIB
       - picoIPv4MIB
       - picoIPv6MIB
+
+  raritan:
+    walk:
+      - sysUpTime
+      - unitCpuTemp
+      - inletCurrent
+      - inletVoltage
+      - inletActivePower
+      - outletOperationalState
+      - outletCurrent
+      - outletMaxCurrent
+      - outletActivePower
+      - outletWattHours
+      - 1.3.6.1.4.1.13742.4.1.2.2.1.6 # outletVoltage
```

Comment:
We usually have the names, with the oids as a comment.

Reply:
See comment thread earlier where using "outletVoltage" conflicted with another MIB and was causing build errors, which is why I switched to using the raw OID.

Or are you asking for everything to be in a format like this (depending on which variant needs to be listed first)?

  - name # OID
  - OID # name

Reply:
Put everything as the oid then I'd say, with the name as a comment.

Reply:
I changed all Raritan-specific names to be OIDs instead, but I left the general "sysUpTime" in place unchanged.  This seems to mirror how other devices have been configured in this file, such as ubiquiti_airmax and ddwrt.